openshift-cli/taskUsingClusterResource/octaskresource.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: task-openshift-command-clusterresource
+spec:
+  inputs:
+    resources:
+      - name: clusterdetails
+        type: cluster
+  steps:
+    - name: version
+      image: quay.io/openshift-pipeline/openshift-cli
+      command:
+        - /usr/local/bin/oc
+      args: ["--kubeconfig", "/workspace/${inputs.resources.clusterdetails.name}/kubeconfig", "--context", "${inputs.resources.clusterdetails.name}", "version"]
```

Comment:
With `v0.3.0` release of `tektoncd/pipeline`, you can now "factor" those into `containerTemplate` :angel:

Reply:
(if you use `KUBECONFIG` env for example, with `args`, it's a bit trickier :sweat_smile: )

Reply:
Tried, not working properly, created an issue on pipeline https://github.com/tektoncd/pipeline/issues/811